### PR TITLE
Bug 1143926 - Do not set appearance on activity terminated since topmost...

### PR DESCRIPTION
**Pull request by mikehenrty**
...windowchanged handles that now

### CODE REVIEW

**Comment by try-server-hook** [comment]
[mikehenrty Michael Henretty (mikehenrty)](https://github.com/mikehenrty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7b453e076760)

File: apps/system/js/statusbar.js [hunk 1]
```diff
@@ -194,7 +194,6 @@ var StatusBar = {
     window.addEventListener('appopened', this);
     window.addEventListener('hierarchytopmostwindowchanged', this);
     window.addEventListener('activityopened', this);
-    window.addEventListener('activityterminated', this);
     window.addEventListener('activitydestroyed', this);
```
Comment (alivedise) [inline apps/system/js/statusbar.js]: nit: maybe we could remove this and activityopened as well?

Comment (mikehenrty) [inline apps/system/js/statusbar.js]: We can't unfortunately because we still need to `updateMinimizedStatusBarWidth` here to prevent bug 1088462.

**Comment by try-server-hook** [comment]
[mikehenrty Michael Henretty (mikehenrty)](https://github.com/mikehenrty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=07ddd701d387)

**Pull request merged**